### DISCUSSION TITLE
YAML Anchors and alises

### DISCUSSION THREAD

**Discussion by ralphv** [blakeblackshear/frigate#9730] [General]
Hello,

As far as I can tell, frigate yaml parses doesn't use the standard yaml anchors:

https://smcleod.net/2022/11/yaml-anchors-and-aliases/#:~:text=The%20YAML%20Spec%20defined%20a,used%20to%20reference%20a%20value.

Would be great if we can support that and it should be easy as it is a feature of the parses itself.

This way, we can have reusable sections. One common thing is to have a couple of detect/record reusable section that we can use for a different cams without copying pasting so much.

**Comment by numericOverflow** [comment]
Are yaml anchors possible yet here in 2025?
